### PR TITLE
adding poliastro

### PR DESCRIPTION
**Pull request by esc**

### CODE REVIEW

**Comment by astrojuanlu** [comment]
Thanks @esc :heart::rocket: I suspect that the failing tests are because recent updates in NumPy and Plotly that were not properly prevented with upper bounds in the poliastro feedstock. I opened an issue upstream https://github.com/conda-forge/poliastro-feedstock/issues/30.

**Comment by astrojuanlu** [comment]
I will try to fix these issues in the upcoming poliastro 0.13.1 release, due this month.

**Comment by esc** [comment]
going to rebase and re-roll this one.

**Comment by esc** [comment]
@Juanlu001 I am picking up these tests again, since we refactored large parts of Numba here:

https://github.com/numba/numba/pull/5197

I now get:

```
============= 9 failed, 365 passed, 3 xfailed, 1 warning in 32.90s =============
```

**Comment by astrojuanlu** [comment]
Thanks @esc and sorry for the delay! I finally fixed the 0.13.1 package and we should be in a better position now.

**Comment by esc** [comment]
@Juanlu001 sounds good. Numba is going through a big refactoring now, so do expect failures from our side at the moment also.

File: switchboard.py [hunk 1]
```diff
@@ -259,5 +259,33 @@ def test_command(self):
...
+        return(git_ls_remote_tags(self.clone_url)[-1])
+
+    @property
+    def conda_dependencies(self):
+        return ["astropy matplotlib numba numpy pandas plotly scipy",
+                "-c conda-forge jplephem astroquery"]
+
+    @property
+    def install_command(self):
+        return "python setup.py install"
```
Comment (astrojuanlu) [inline switchboard.py:283]:
```suggestion
        return "python -m pip install ."
```

Or perhaps `flit install`?

**Comment by astrojuanlu** [comment]
Preparing conda-forge packages for poliastro 0.14.0!

**Comment by epassaro** [comment]
@astrojuanlu are you looking for help with this?

**Comment by astrojuanlu** [comment]
Hey! I thought we had merged this :) Sure @epassaro , do you want to give this a spin? Also, poliastro 0.15.0 will be out this Friday, hopefully.

**Comment by esc** [comment]
Yes, sorry, I dropped the ball on this, @epassaro please feel free to look into this. I am happy to help getting it over the line and merging it too! 🙌

**Comment by esc** [comment]
it may be worth rebasing this, btw.

**Comment by esc** [comment]
closing in favour of #77

**Pull request closed without merging**